Any idea how to get xxhash working on a windows CI/CD?

In this PR I was getting

> D:\a\llama.cpp\llama.cpp\examples\gguf-hash\deps\xxhash\xxhash.h(1691,1): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory [D:\a\llama.cpp\llama.cpp\build\examples\gguf-hash\xxhash.vcxproj]

But can't seem to figure out a solution for it. Attempted this [workaround](https://github.com/ggerganov/llama.cpp/pull/8048/commits/ddfe234ccc03afbf86beb0a33101f9d47c7967c2) but that didn't solve the problem.

https://github.com/ggerganov/llama.cpp/pull/8048

--- a/xxhash.h
+++ b/xxhash.h
@@ -1688,8 +1688,7 @@ struct XXH64_state_s {
 #ifndef XXH_NO_XXH3
 
 #if defined(__STDC_VERSION__) && (__STDC_VERSION__ >= 201112L) /* >= C11 */
-#  include <stdalign.h>
-#  define XXH_ALIGN(n)      alignas(n)
+#  define XXH_ALIGN(n)      _Alignas(n)
 #elif defined(__cplusplus) && (__cplusplus >= 201103L) /* >= C++11 */
 /* In C++ alignas() is a keyword */
 #  define XXH_ALIGN(n)      alignas(n)

